src/edu/csus/ecs/pc2/util/ScoreboardVariableReplacer.java

```diff
@@ -0,0 +1,131 @@
+// Copyright (C) 1989-2021 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.util;
+
+import edu.csus.ecs.pc2.core.CommandVariableReplacer;
+import edu.csus.ecs.pc2.core.model.Account;
+import edu.csus.ecs.pc2.core.model.ClientId;
+import edu.csus.ecs.pc2.core.model.Group;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+
+/**
+ * Substitute scoreboard team values for variables.
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ *
+ */
+public class ScoreboardVariableReplacer {
+
+    public static final String COUNTRY_CODE = "{:countrycode}";
+
+    public static final String LONG_SCHOOL_NAME = "{:longschoolname}";
+
+    public static final String SHORT_SCHOOL_NAME = "{:shortschoolname}";
+
+    public static final String GROUP_NAME = "{:groupname}";
+
+    public static final String GROUP_ID = "{:groupid}";
+
+    public static final String EXTERNAL_ID = "{:externalid}";
+
+    public static final String CLIENT_NUMBER = "{:clientnumber}";
+
+    public static final String TEAM_LOGIN_NAME = "{:teamloginname}";
+
+    public static final String SITE_NUMBER = "{:sitenumber}";
+
+    public static final String TEAM_NAME = "{:teamname}";
+
+    public static final String[] VARIABLE_NAMES = { //
+            TEAM_NAME, //
+            SITE_NUMBER, //
+            TEAM_LOGIN_NAME, //
+            CLIENT_NUMBER, //
+            EXTERNAL_ID, //
+            GROUP_ID, //
+            GROUP_NAME, //
+            SHORT_SCHOOL_NAME, //
+            LONG_SCHOOL_NAME, //
+            COUNTRY_CODE, //
+    };
+
+    /**
+     * Substitutes values for variables.
+     * 
+     * 
+     * Variable names and examples.
+     * 
+     * <pre>
+     * Client/Team number - {:clientnumber} = 514
+     * Country Code - {:countrycode} = CAN
+     * CMS/External ID - {:externalid} = 309407
+     * Group Id Number - {:groupid} = 12545
+     * Group name - {:groupname} = Canada - University of British Columbia D1
```

Comment:
This does not seem to me to be a good example of a `Group name` -- this is a TEAM name.  A `Group name` would be, like `PacNW D1` or `Graduate Division`, or similar.

Reply:
no change.